identity/b64json.go

```diff
@@ -0,0 +1,30 @@
+package identity
```

Comment:
Not sure that this file belongs in identity, might be better to put it in separate package

Reply:
Done